tests/conftest.py

```diff
@@ -37,17 +51,17 @@ def restore_config(request, node_config):
 
 @pytest.fixture(scope='module')
 def node_config():
-    return copy.deepcopy(CONFIG)
+    return copy.deepcopy(CONFIG_ED25519)
```

Comment:
Does this mean that we are only going to test with `ED25519` and not `ECDSA`?
If so we should decide right now that we are only going to use `ed25519` and remove support for `ecdsa` in `crypto.py`